doc/developing-packages.rst

```diff
@@ -4,52 +4,57 @@ Quickstart
 .. TIP::
     If this is your first time using `cabal` you should check out the `Getting Started guide <getting-started.html>`__.
 
-Let's assume we have created a project directory and already have a
-Haskell module or two.
+Starting from scratch, we're going to walk you through creating a simple
+Haskell application.
 
-Every project needs a name, we'll call this example "proglet".
+**TL;DR;** ``mkdir proglet && cd proglet && cabal init && cabal run proglet``
```

Comment:
```suggestion
**TL;DR;** ``mkdir proglet && cd proglet && cabal init --simple --exe && cabal run proglet``
```

@ptkato - this fails on the following: 

```haskell
λ athena proglet → cabal run proglet
Resolving dependencies...
Build profile: -w ghc-8.10.4 -O1
In order, the following will be built (use -v for more details):
 - proglet-0.1.0.0 (exe:proglet) (first run)
Configuring executable 'proglet' for proglet-0.1.0.0..
Preprocessing executable 'proglet' for proglet-0.1.0.0..
Building executable 'proglet' for proglet-0.1.0.0..
[1 of 1] Compiling Main             ( app/Main.hs, /home/tapas/haskell/proglet/dist-newstyle/build/x86_64-linux/ghc-8.10.4/proglet-0.1.0.0/x/proglet/build/proglet/proglet-tmp/Main.o )

app/Main.hs:1:8: error:
    Could not load module ‘Prelude’
    It is a member of the hidden package ‘base-4.14.1.0’.
    Perhaps you need to add ‘base’ to the build-depends in your .cabal file.
    Use -v (or `:set -v` in ghci) to see a list of the files searched for.
  |
1 | module Main where
  |        ^^^^
```

It looks like the dependency heuristic is not finding `base` in the `exe` case for `Simple` type builds. Do you mind fixing?

Reply:
Fixed in #7552. You can now run the following and it will build with output: 

```haskell
mkdir proglet && cd proglet && cabal init --simple --exe && cabal run proglet
```

Reply:
Done (both doc tweak and approval of PR).